design/ErrorHandling.md

```diff
@@ -288,6 +288,7 @@ typedef (GPUOutOfMemoryError or GPUValidationError) GPUError;
 
 partial interface GPUDevice {
     void pushErrorScope(GPUErrorFilter filter);
+    // Rejects with DOMException("OperationError") if there are no error scopes on the stack.
```

Comment:
Somehow I missed this the first time, can you change this to say "Throws OperationError" instead of "Rejects"?

Reply:
Removed it because it stated in the text above.